docs/Using-Fleet/MDM-migration-guide.md

```diff
@@ -27,7 +27,14 @@ This section provides instructions for migrating your hosts away from your old M
     2. In ABM, assign these hosts' MDM server to Fleet: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Assign to the following MDM:**, select your Fleet server in the dropdown, and select **Continue**.
 5. In your old MDM solution, unenroll the hosts to be migrated. MacOS does not allow multiple MDMs to be installed at once.
 6. Send [these guided instructions](#instructions-for-end-users) to your end users to complete the final few steps via Fleet Desktop.
-    * Note that there will be a gap in MDM coverage between when the host is unenrolled from the old MDM and when the host turns on MDM in Fleet.
+    * Note that there will be a gap in MDM coverage between when the host is unenrolled from the old
+      MDM and when the host turns on MDM in Fleet.
+
+You can also optionally set a dark and light background logo that the end user will see on their
+device during the completion of their migration. This can be set by going to **Settings** > **Organization settings** > **Organization info** and adding a url to the desired image in the
+**Organization avatar URL (for dark backgrounds)** and **Organization avatar URL (for light
+backgrounds)** inputs. These images will show depending on the system theme that the
+user is running on their device during the end user migration flow.
```

Comment:
@noahtalerman are you happy with these docs for the new light/dark feature? if so I'll merge

Reply:
Hey @ghernandez345 I haven't had the chance to check these changes out yet. I will review post standup today.